Stackdriver-REST-API/README.md

```diff
@@ -0,0 +1,113 @@
+# Catchpoint and Google's Cloud Monitoring & Logging
+
+## Introduction
+
+Cloud Monitoring with Catchpoint provides visibility into cloud services and the applications they depend on. Cloud services depend on many different services, from Nginx to Elasticsearch to Cassandra. With Cloud Monitoring, Operations teams can proactively monitor their infrastructure with data collected from application instrumentation, probes, events, and metadata. With integrations to collaboration tools, alerts can be sent via Slack, PagerDuty, and others. And with advanced charting and dashboards, analysts can isolate issues and recover quickly.
+
+This integration uses the [Cloud Monitoring](https://cloud.google.com/monitoring) REST API to ingest raw data from Catchpoint.
+
+### Catchpoint Tests
+
+#### Supported Metrics
+
+1. **Connect** The time to establish a connection with a specific URL. Reported in milliseconds.
```

Comment:
Should the list of metrics and test types be the same for both methods?

Reply:
We have tried to show the integration can be done in both ways and the REST API method can be customized as per the use case.
Webhook method is recommended/preferred for this Integrations.